src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -56,13 +57,12 @@ class ClientCredentials(object):
   functions.
   """
 
-  def __init__(self, low_credentials, intermediary_low_credentials):
+  def __init__(self, low_credentials):
     self._low_credentials = low_credentials
-    self._intermediary_low_credentials = intermediary_low_credentials
 
 
-def ssl_client_credentials(root_certificates, private_key, certificate_chain):
-  """Creates a ClientCredentials for use with an SSL-enabled Channel.
+def ssl_channel_credentials(root_certificates, private_key, certificate_chain):
```

Comment:
Assuming that the default roots in the python/core package are installed/configured correctly (and we have a challenge with that right now, see #4224), then everything set to `None` is indeed the 90% use case (a grpc client connecting to a server with a publicly trusted cert).

Also , it does not make sense to have a private key with a cert chain and vice-versa so maybe these could be a tuple but I'll leave that to your python sense of zen :).